desktop/cmp/grouping/GroupingChooser.ts

```diff
@@ -51,7 +50,7 @@ export const [GroupingChooser, groupingChooser] = hoistCmp.withFactory({
             ...layoutProps,
             item: popover({
                 isOpen,
-                popoverRef: (v) => model.popoverRef(v), // Workaround for #2272
+                popoverRef: (v) => (model.popoverRef as any)(v), // Workaround for #2272
```

Comment:
Not sure if this is the best way to do this, but Typescript wasn't happy with calling methods on the ref

Reply:
I can look at the type we are passing back for createObservableRef now.  Maybe it needs to be improved.  Note that it is generic, so you can say indicate what type your Ref is going to eventually refer to/receive.